python/vyos/util.py

```diff
@@ -903,3 +902,34 @@ def check_port_availability(ipaddress, port, protocol):
         return True
     except:
         return False
+
+def install_into_config(config_paths, override_prompt=True):
+    # Allows op-mode scripts to install values if called from an active config session
+    # config_paths: dict of config paths
+    # override_prompt: if True, user will be prompted before existing nodes are overwritten
+
+    if not config_paths:
+        return None
+
+    from vyos.config import Config
+
+    conf = Config()
+
+    if not conf.in_session():
+        print('You are not in configure mode, commands to install manually from configure mode:')
+        for path in config_paths:
+            print(f'set {path}')
+        return None
+
+    count = 0
+
+    for path in config_paths:
+        if override_prompt and conf.exists(path) and not conf.is_multi(path):
```

Comment:
Can we use an instance of `ConfigTreeQuery` for `conf`?

Reply:
As the underlying code essentially uses the `Config` class and we needed an instance for the session check, it seemed more efficient to use just the single instance for both checks.

Reply:
Please still use ConfigTreeQuery whenever possible from op mode.